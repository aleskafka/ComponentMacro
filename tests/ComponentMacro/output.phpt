<?php

/**
 * Test: Latte\Engine: {include file}
 */

require __DIR__ . '/../bootstrap.php';

$latte = new Latte\Engine;
$latte->setTempDirectory(TEMP_DIR);
ComponentMacro::install($latte, __DIR__ . '/components');

matchFile(
	__DIR__ . '/expected/output.html',
	$latte->renderToString(__DIR__ . '/templates/output.latte', ['title' => 'Hello', 'navigation' => ['John']])
);

