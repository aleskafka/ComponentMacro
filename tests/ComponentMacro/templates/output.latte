
<Module n:component>
	<ModuleTitle n:component><h2>Hello World</h2></ModuleTitle>
	<ModuleBody n:component>body</ModuleBody>
</Module>
