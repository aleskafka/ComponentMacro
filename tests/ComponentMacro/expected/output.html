
<div class="module">
	<div class="module-wrapper">
		<div class="module-title"><h2>Hello World</h2></div>
		<div class="module-body">body</div>
	</div>
</div>
